components/filter/filter.js

```diff
@@ -361,6 +376,15 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		this._setFilterCounts();
 	}
 
+	_isDimensionEmpty(dimension) {
```

Comment:
This will eventually be used by the search input and clear button as well, and be a tad more complicated comparison once search is in